frontend/www/js/omegaup/components/qualitynomination/List.vue

```diff
@@ -1,106 +1,195 @@
 <template>
-  <div class="panel panel-default">
-    <div class="panel-heading">
-      <h3 class="panel-title">
+  <div>
+    <div class="form-inline" v-if="!myView">
+      <select name="column" class="form-control" v-model="selectColumn">
+        <option
+          v-for="(columnText, columnIndex) in columns"
+          v-bind:value="columnIndex"
+        >
+          {{ columnText }}</option
+        >
+      </select>
+      <omegaup-autocomplete
+        v-bind:init="el => typeahead.problemTypeahead(el)"
+        v-model="query"
```

Comment:
```suggestion
        v-model="queryProblem"
```

por consistencia con L22.

Reply:
listo